docs/faqs-and-tips.md

```diff
@@ -53,6 +53,16 @@ RCT_EXPORT_METHOD(crash) {
 ```
 Just disable your debugger in Xcode. 'Project name' -> 'Edit Scheme...' -> 'Run' -> deselect "Debug executable"
 
+### I have the latest SDK installed, but I can't send a test messages from the console. How do I fix this?
+
+Sometimes when building a message in the console, sending a test to a device is not possible, as the "Test on device" button is grayed out. This can be very annoying, but have found a "work around" that enables the button:
```

Comment:
```suggestion
Sometimes when building an in-app-message in the console, sending a test to a device is not possible, as the "Test on device" button is grayed out. This can be very annoying, but have found a "work around" that enables the button:
```